requirements.txt

```diff
@@ -25,21 +25,21 @@ m2r==0.3.1
 mako==1.2.4 ; python_version >= '3.7'
 markdown-it-py==2.1.0 ; python_version >= '3.7'
 markupsafe==2.1.2 ; python_version >= '3.7'
-matplotlib==3.6.3 ; python_version >= '3.8'
+matplotlib==3.7.0 ; python_version >= '3.8'
 mccabe==0.7.0 ; python_version >= '3.6'
-mdit-py-plugins==0.3.3 ; python_version >= '3.7'
+mdit-py-plugins==0.3.4 ; python_version >= '3.7'
```

Comment:
Do we use this to parse our markdown documentation?
If so where do we use that?